src/subscription_manager/managercli.py

```diff
@@ -1145,10 +1144,6 @@ def _do_command(self):
                 print(_("Identity certificate has been regenerated."))
 
                 log.debug("Successfully generated a new identity from server.")
-        except connection.GoneException as ge:
```

Comment:
Is it possible that this will still be needed?

Reply:
It is, I'll add it back and squash